pkg/receiver/telegrafreceiver/README.md

```diff
@@ -55,3 +55,82 @@ With its current implementation Telegraf receiver has the following limitations:
 - only the following Telegraf metric data types are supported:
   - `telegraf.Gauge` that is translated to `pdata.MetricDataTypeGauge`,
   - `telegraf.Counter` that is translated to `pdata.MetricDataTypeSum`.
+
+## Migration from Telegraf
+
+### Data model
+
+Internal OTC metric format differs from the Telegraf one and `separate_field` controls the conversion:
+
+- If `separate_field` is `false`, the Open Telemetry metric name is going to be concatenated from the Telegraf metric name
+  and the Telegraf field with `_` as separator.
+- If `separate_fields` is `true`, the Open Telemetry metric name is going to be the same as the Telegraf one,
+  and the Telegraf `field` is going to be converted to the Open Telemetry attribute.
```

Comment:
not attribute itself but data point label

Reply:
even data point attribute

```
Metric #4
Descriptor:
     -> Name: tomcat_jmx_jvm_memory
     -> Description: 
     -> Unit: 
     -> DataType: Gauge
NumberDataPoints #0
Data point attributes:
     -> field: STRING(NonHeapMemoryUsage.init)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2022-03-10 11:28:30 +0000 UTC
Value: 2555904.000000
```